security/mobile-app-network-security.md

```diff
@@ -0,0 +1,126 @@
+# Abstract
+
+In this document we will discuss some of the common problems around network security for mobile developers consuming AeroGear services in their apps, and propose a solution that will help resolve those problems.
+
+We will first look at some of the use cases in order to define the problems, and then look at some of the options to solve them. Finally the proposed solution will be presented.
+
+# Goals
+
+## Primary Goal
+
+* Reach agreement on a solution that can help mobile developers properly handle network security in their mobile apps that consume AeroGear services
```

Comment:
You can probably remove 'Reach agreement on' as this PR review is what gets the agreement. 
Just have 'A solution that ....'